identity-admin/webapp/pages/api/users.ts

```diff
@@ -0,0 +1,67 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { SearchResults, User } from '../../interfaces';
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse<SearchResults>
+) => {
```

Comment:
Missing return type here:

```suggestion
): void => {
```